contents/ops/ops.qmd

```diff
@@ -113,7 +113,11 @@ Learn more about ML Lifecycles through a case study featuring speech recognition
 
 In this chapter, we will provide an overview of the core components of MLOps, an emerging set of practices that enables robust delivery and lifecycle management of ML models in production. While some MLOps elements like automation and monitoring were covered in previous chapters, we will integrate them into a framework and expand on additional capabilities like governance. Additionally, we will describe and link to popular tools used within each component, such as [LabelStudio](https://labelstud.io/) for data labeling. By the end, we hope that you will understand the end-to-end MLOps methodology that takes models from ideation to sustainable value creation within organizations.
 
-### Data Management
+@fig-ops-layers shows the MLOps system stack. The MLOps lifecycle starts from data management and CI/CD pipelines for model development. Developed models go through model training and evaluation. Once trained to convergence, model deployment brings models up to production and ready to serve. After deployment, model serving reacts to workload changes and meets service level agreements cost-effectively when serving millions of end users or AI applications. Infrastructure management ensures the necessary resources are available and optimized throughout the lifecycle. Continuous monitoring, Governance, and communication and collaboration are the remaining pieces of MLOps to ensure seamless development and operations of ML models.
```

Comment:
nit: "Governance" -> "governance" in the below sentence:
> Continuous monitoring, Governance, and communication and collaboration...

Reply:
Thank you for catching this!